rubies/ruby/CVE-2015-9096.yml

```diff
@@ -17,3 +17,4 @@ description: |
   (Terada, p. 4) can be applied to without any modification.
 patched_versions:
   - ">= 2.4.0"
+  - ">= 2.3.5"
```

Comment:
I think this should be `"~> 2.3.5"` since we already have a greater equals for 2.4.0.

Reply:
Sorry. I fixed.